data_steward/tools/consolidated_achilles_report/get_all_results_html.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import print_function
+import gcs_utils
```

Comment:
According to [pep 8](https://www.python.org/dev/peps/pep-0008/#imports), your imports should be ordered as:

```
import json
import os

import gcs_utils
import query_reports
```

I like to add comments in to help remind me to do this.  Something like:
```
# Python imports
import json
import os

# Third party imports

# Project imports
import gcs_utils
import query_reports
```